mlir/lib/Support/Passes.cpp

```
@@ -10,19 +10,64 @@
+  addSimplificationPasses(passManager);
+  passManager.addPass(qtensor::createShrinkQTensorToFitPass());
+  passManager.addPass(createRemoveDeadValuesPass());
+}
+
+void populateQIRCleanupPipeline(PassManager& passManager) {
+  addSimplificationPasses(passManager);
+  passManager.addPass(qir::createQIRCleanupPass());
+  passManager.addPass(createRemoveDeadValuesPass());
+}
+
+void runQCCleanupPipeline(ModuleOp module) {
```

Cppcheck (2.20.0)

[style] The function 'runQCCleanupPipeline' is never used.

(unusedFunction)